EntranceShuffle.py

```diff
@@ -859,7 +863,7 @@ def validate_world(world, worlds, entrance_placed, locations_to_ensure_reachable
             if impas_front_entrance is not None and impas_back_entrance is not None and not same_hint_area(impas_front_entrance, impas_back_entrance):
                 raise EntranceShuffleError('Kak Impas House entrances are not in the same hint area')
 
-    if (world.shuffle_special_interior_entrances or world.settings.shuffle_overworld_entrances or world.settings.spawn_positions) and \
+    if (world.shuffle_special_interior_entrances or world.settings.shuffle_overworld_entrances or (world.settings.spawn_positions == ['child', 'adult'])) and \
```

Comment:
Let the no item search run if any spawn shuffle is on, not just both. ER isn't aware of starting age.

Reply:
yeah I actually just noticed this and already fixed it so it'll be in the next push